.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: node_js
+node_js:
+ - '10.1.0'
+sudo: disabled
+
+cache: 
+  directories:
+    - "node_modules"
+
+
+install:
+  - npm install
+
+before_script:
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then ./node_modules/.bin/precise-commits --whitelist="./**/!(*.json)" --check-only --head=$TRAVIS_PULL_REQUEST_SHA --base=$(git merge-base HEAD $TRAVIS_BRANCH); fi'
```

Comment:
Add a comment above this line to indicate what you're doing.  I know what this is, but people in the future might wonder.